torch/csrc/jit/script/compiler.cpp

```diff
@@ -429,6 +429,12 @@ struct Environment {
           {"all", std::make_shared<BuiltinFunction>(aten::all, at::nullopt)},
           {"list", std::make_shared<BuiltinFunction>(aten::list, at::nullopt)},
           {"ord", std::make_shared<BuiltinFunction>(aten::ord, at::nullopt)},
+          {"chr", std::make_shared<BuiltinFunction>(aten::chr, at::nullopt)},
+          {"hex", std::make_shared<BuiltinFunction>(aten::hex, at::nullopt)},
```

Comment:
`round` `hex` and `oct` are magic methods, and round has a second argument for number of decimal places that defaults to 0

https://rszalski.github.io/magicmethods/

Reply:
@eellison I noticed the second optional argument for `round` but it has a weird behavior that is hard to match from cpp side. See notes in https://docs.python.org/3/library/functions.html#round. 
Also I think round is rarely with that arguments, would prefer to delay implementing it unless it's necessary. :P

Reply:
`round` with optional args's impl in cpython is a bit long. I will make that a low pri item for following patches.

Reply:
`round` without arguments is the same thing as `int`, so there's not much utility in having it without the argument.

Reply:
@eellison They're different :P 
```
round(2.6)=3
int(2.6)=2
```